server.js

```diff
@@ -1014,6 +1014,31 @@ function setAnchors() {
     }
 }
 
+function removeAvatars() {
+    let objectsToDelete = [];
+
+    // load all object folders
+    let tempFiles = fs.readdirSync(objectsPath).filter(function (file) {
+        return fs.statSync(path.join(objectsPath, file)).isDirectory();
+    });
+    // remove hidden directories
+    while (tempFiles.length > 0 && tempFiles[0][0] === '.') {
+        tempFiles.splice(0, 1);
+    }
+
+    // populate all objects folders with object.json files.
+    tempFiles.forEach(function (objectKey) {
+        if (objectKey.includes('_AVATAR_')) {
+            objectsToDelete.push(objectKey);
+        }
+    });
+    
+    objectsToDelete.forEach(objectKey => {
```

Comment:
Based on this using rmSync over the deleteObjects API it feels like this would be a bit clearer named removeAvatarFiles